grr/server/grr_response_server/gui/ui/components/client_details/module.ts

```diff
@@ -11,6 +12,8 @@ import {HumanReadableSizeModule} from '../human_readable_size/module';
 import {TimestampModule} from '../timestamp/module';
 
 import {ClientDetails} from './client_details';
+import {EntryHistoryButton} from './entry_history_button/entry_history_button';
+import {EntryHistoryDialog} from './entry_history_dialog/entry_history_dialog';
```

Comment:
Create an own module for EntryHistoryButton/Dialog.

Reply:
I believe it would be better to keep them inside the `ClientDetailsModule`, as they are pretty related and not used elsewhere. Also, in the next PR there are 3 more components that are used in the `ClientDetails`, as well as in the `EntryHistoryDialog`. Where would they fit?

Reply:
Our build system really pushes to having one module per folder. It's your choice to either keep EntryHistoryDialog in the sub-folder and add a module, or move the files one folder up and have them in the parent module.